tools/regenerate-lockfile-fixtures.ts

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env ts-node
+
+/**
+ * Utility script to regenerate lockfile fixtures for template tests.
+ *
+ * Run this script when:
+ * - Webpack or Vite versions are updated in the main Forge package.json
+ * - Template dependencies change significantly
+ * - Yarn lockfile format changes
+ * - Template tests start failing due to outdated lockfiles
+ *
+ * Usage:
+ *   yarn ts-node tools/regenerate-lockfile-fixtures.ts
+ *   or
+ *   yarn update:lockfile-fixtures (if added to package.json scripts)
+ */
+
+import * as fs from 'node:fs/promises';
+import * as path from 'node:path';
+
+import { api } from '../packages/api/core/src/api/index';
+import {
+  PACKAGE_MANAGERS,
+  spawnPackageManager,
+} from '../packages/utils/core-utils/src/index';
+import * as testUtils from '../packages/utils/test-utils/src/index';
+
+async function ensureDirectoryExists(dir: string) {
+  await fs.mkdir(dir, { recursive: true });
+}
+
+async function regenerateWebpackTypescriptLockfile() {
+  console.log('Regenerating webpack-typescript lockfile fixture');
+
+  const dir = await testUtils.ensureTestDirIsNonexistent();
+
+  try {
+    await api.init({
+      dir,
+      template: path.resolve(
+        __dirname,
+        '..',
+        'packages',
+        'template',
+        'webpack-typescript',
+      ),
+      interactive: false,
+    });
+
+    const packageJsonPath = path.join(dir, 'package.json');
+    const pj = JSON.parse(await fs.readFile(packageJsonPath, 'utf-8'));
+
+    const webpackPackageJson = JSON.parse(
+      await fs.readFile(
+        path.resolve(
+          __dirname,
+          '..',
+          'node_modules',
+          'webpack',
+          'package.json',
+        ),
+        'utf-8',
+      ),
+    );
+    const webpackVersion = webpackPackageJson.version;
+    pj.resolutions = {
+      webpack: webpackVersion,
+    };
+
+    await fs.writeFile(packageJsonPath, JSON.stringify(pj, null, 2));
+
+    console.log(`  Added webpack resolution: ${webpackVersion}`);
+
+    console.log('  Running yarn install...');
+    await spawnPackageManager(PACKAGE_MANAGERS['yarn'], ['install'], {
+      cwd: dir,
+      env: {
+        ...process.env,
+        YARN_ENABLE_HARDENED_MODE: '0',
+        YARN_ENABLE_IMMUTABLE_INSTALLS: '0',
```

Comment:
Can we remove these flags now that the expectation is that the developer runs these scripts locally?